[2025-07-17T12:06:31.956+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-07-17T12:06:32.149+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: bank_reviews_etl_and_analysis_pipeline.insert_data_to_postgresql manual__2025-07-17T12:03:05.725692+00:00 [queued]>
[2025-07-17T12:06:32.163+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: bank_reviews_etl_and_analysis_pipeline.insert_data_to_postgresql manual__2025-07-17T12:03:05.725692+00:00 [queued]>
[2025-07-17T12:06:32.165+0000] {taskinstance.py:2867} INFO - Starting attempt 1 of 2
[2025-07-17T12:06:32.192+0000] {taskinstance.py:2890} INFO - Executing <Task(PythonOperator): insert_data_to_postgresql> on 2025-07-17 12:03:05.725692+00:00
[2025-07-17T12:06:32.202+0000] {standard_task_runner.py:72} INFO - Started process 2488 to run task
[2025-07-17T12:06:32.206+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'bank_reviews_etl_and_analysis_pipeline', 'insert_data_to_postgresql', 'manual__2025-07-17T12:03:05.725692+00:00', '--job-id', '64', '--raw', '--subdir', 'DAGS_FOLDER/dag_load_reviews.py', '--cfg-path', '/tmp/tmpszbj0qzh']
[2025-07-17T12:06:32.209+0000] {standard_task_runner.py:105} INFO - Job 64: Subtask insert_data_to_postgresql
[2025-07-17T12:06:32.272+0000] {task_command.py:467} INFO - Running <TaskInstance: bank_reviews_etl_and_analysis_pipeline.insert_data_to_postgresql manual__2025-07-17T12:03:05.725692+00:00 [running]> on host 569ba507bc01
[2025-07-17T12:06:32.396+0000] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='mohammed' AIRFLOW_CTX_DAG_ID='bank_reviews_etl_and_analysis_pipeline' AIRFLOW_CTX_TASK_ID='insert_data_to_postgresql' AIRFLOW_CTX_EXECUTION_DATE='2025-07-17T12:03:05.725692+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-07-17T12:03:05.725692+00:00'
[2025-07-17T12:06:32.399+0000] {logging_mixin.py:190} INFO - Task instance is in running state
[2025-07-17T12:06:32.401+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: queued
[2025-07-17T12:06:32.403+0000] {logging_mixin.py:190} INFO - Current task name:insert_data_to_postgresql state:running start_date:2025-07-17 12:06:32.151830+00:00
[2025-07-17T12:06:32.404+0000] {logging_mixin.py:190} INFO - Dag name:bank_reviews_etl_and_analysis_pipeline and current dag run status:running
[2025-07-17T12:06:32.406+0000] {taskinstance.py:732} INFO - ::endgroup::
[2025-07-17T12:06:32.411+0000] {logging_mixin.py:190} INFO - ✅ Connexion réussie à la base de données Aiven PostgreSQL.
[2025-07-17T12:06:32.666+0000] {logging_mixin.py:190} INFO - ❌ Erreur PostgreSQL : relation "public.all_bank_reviews" does not exist
[2025-07-17T12:06:32.672+0000] {python.py:240} INFO - Done. Returned value was: None
[2025-07-17T12:06:32.688+0000] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-07-17T12:06:32.690+0000] {taskinstance.py:353} INFO - Marking task as SUCCESS. dag_id=bank_reviews_etl_and_analysis_pipeline, task_id=insert_data_to_postgresql, run_id=manual__2025-07-17T12:03:05.725692+00:00, execution_date=20250717T120305, start_date=20250717T120632, end_date=20250717T120632
[2025-07-17T12:06:32.738+0000] {logging_mixin.py:190} INFO - Task instance in success state
[2025-07-17T12:06:32.740+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: running
[2025-07-17T12:06:32.741+0000] {logging_mixin.py:190} INFO - Dag name:bank_reviews_etl_and_analysis_pipeline queued_at:2025-07-17 12:03:06.860803+00:00
[2025-07-17T12:06:32.743+0000] {logging_mixin.py:190} INFO - Task hostname:569ba507bc01 operator:PythonOperator
[2025-07-17T12:06:32.785+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-07-17T12:06:32.828+0000] {taskinstance.py:3901} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-07-17T12:06:32.838+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
